sources/engine/Stride.Engine/Engine/Splines/Models/BoundingBoxMesh.cs

```diff
@@ -0,0 +1,70 @@
+//// Copyright (c) Stride contributors (https://Stride.com)
+//// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Stride.Core.Mathematics;
+using Stride.Graphics;
+using Stride.Rendering;
+using Buffer = Stride.Graphics.Buffer;
+
+namespace Stride.Engine.Splines.Models
+{
+    public class BoundingBoxMesh
+    {
+        public MeshDraw MeshDraw;
```

Comment:
Consider making this mesh private so that it can't be overwritten from outside this class and exposing it through a get-only property.